qmxgraph/widget.py

```diff
@@ -329,6 +470,17 @@ def toggle_inspector(self):
         else:
             self.show_inspector()
 
+    def remove_cells(self, cell_ids, *, ignore_missing_cells=False):
+        """
+        Remove cells from graph.
+
+        :param list cell_ids: Ids of cells that must be removed.
+        :param bool ignore_missing_cells: Ids of non existent cells are
+            ignored instead raising an error.
+        """
+        with wait_signals_called(self._events_bridge.on_cells_removed):
```

Comment:
We need this or is it a left over from before the api calls "wait" for completion?

Reply:
We still need this I believe.